x-pack/plugins/triggers_actions_ui/public/application/lib/rule_api/aggregate.ts

```diff
@@ -29,15 +29,15 @@ export async function loadRuleAggregations({
   searchText,
   typesFilter,
   actionTypesFilter,
-  ruleStatusesFilter,
+  ruleExecutionStatusesFilter,
 }: {
   http: HttpSetup;
   searchText?: string;
   typesFilter?: string[];
   actionTypesFilter?: string[];
-  ruleStatusesFilter?: string[];
+  ruleExecutionStatusesFilter?: string[];
 }): Promise<RuleAggregations> {
-  const filters = mapFiltersToKql({ typesFilter, actionTypesFilter, ruleStatusesFilter });
```

Comment:
@JiaweiWu Out of curiosity I was wondering why the new ruleStatusesFilter is not included here.

Reply:
do we want to filter the aggregate results as well? I suppose we do, I can add it